cms/templates/import.html

```diff
@@ -127,6 +185,19 @@ <h3 class="title">${_("Success")}</h3>
       </form>
     </article>
 
+  %if library:
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+          <h3 class="title-3">${_("Why import a library?")}</h3>
+          <p>${_("You may want to run a new version of an existing library, or replace an existing library altogether. Or, you may have developed a library outside Studio.")}</p>
```

Comment:
@Kelketek Suggested rewrite:
"You might want to update an existing library to a new version, or replace an existing library entirely. You might also have developed a library outside of Studio." @marcotuts  OK?